core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -196,11 +212,50 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        return !exoPlayer.getSubtitleTracks().isEmpty();
+    }
+
+    @Override
+    public void selectSubtitleTrack(PlayerSubtitleTrack subtitleTrack, final PlayerSubtitlesView playerSubtitlesView) {
```

Comment:
Instead of passing the view to every method using it should we use a similar approach to the playerView and use attach/detach ? To be consistent with out API